mteb/tasks/Clustering/deu/BlurbsClusteringP2P.py

```diff
@@ -36,7 +43,7 @@ class BlurbsClusteringP2P(AbsTaskClustering):
     )
 
 
-class BlurbsClusteringP2PFast(AbsTaskClustering):
+class BlurbsClusteringP2PFast(AbsTaskClusteringFast):
```

Comment:
Need to update the number of samples in line 82?

Reply:
Done